src/components/Refresh/Refresh.tsx

```diff
@@ -4,13 +4,19 @@ import { Button, MenuItem, Icon, DropdownButton } from 'patternfly-react';
 import { config } from '../../config';
 import { PollIntervalInMs } from '../../types/Common';
 
-type Props = {
+type ComponentProps = {
   id: string;
-  pollInterval?: PollIntervalInMs;
   handleRefresh: () => void;
+};
+
+type ReduxProps = {
+  pollInterval: PollIntervalInMs;
```

Comment:
It is called `pollInterval` because that is what this `Refresh` component calls it and I didn't want to go renaming others variable names until we have agreed on a standard. This occurs in many places BTW.